src/MercadoLibre/Provider.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace SocialiteProviders\MercadoLibre;
+
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
```

Comment:
Could you add an `@see` docblock with a link to the provider's docs here please?